### PR TITLE
fix: byoc -> ccs.enabled

### PR DESCRIPTION
**Pull request by psturc**
### JIRA
https://issues.redhat.com/browse/MGDAPI-382

### Notes
* updated cluster template
* updated docs
* I haven't changed any variables in order to keep the backward compatibility with our pipelines and for other teams that are using this tooling (e.g. RHI)

### Verification
https://master-jenkins-csb-intly.cloud.paas.psi.redhat.com/job/ManagedAPI/job/managed-api-install-master/30/console

### CODE REVIEW

File: docs/ocm/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ make ocm/login
 
 **BYOC**
```
Comment (austincunningham) [inline docs/ocm/README.md:25]: maybe update this 
```suggestion
**BYOC/CCS**
```

File: templates/ocm/cluster-template.json [hunk 2]
```diff
@@ -1,6 +1,8 @@
 {
-    "byoc": false,
     "display_name": "long_cluster_name",
+    "ccs": {
+      "enabled": false
+    },
```
Comment (austincunningham) [inline templates/ocm/cluster-template.json:5]: Are the minimum number nodes changing also with this change?

> Minimum worker nodes reduced from 4 to 2 for a single AZ CCS cluster
Minimum worker nodes reduced from 9 to 3 for a multi AZ CCS cluster

File: docs/ocm/README.md (hunk 1 above)
Comment (trepel) [reply docs/ocm/README.md:25]: And maybe similar change in **Multiple AZ** section as well

**Comment by trepel** [comment]
LGTM (apart from that cosmetic readme modifications), no idea what the answer for Austins question is though.

File: templates/ocm/cluster-template.json (hunk 2 above)
Comment (psturc) [reply templates/ocm/cluster-template.json:5]: good point, I've sent an email about this, let's wait for a response

Comment (psturc) [reply templates/ocm/cluster-template.json:5]: Currently it doesn't work with the minimum worker nodes anyway
```
{
  "kind": "Error",
  "id": "400",
  "href": "/api/clusters_mgmt/v1/errors/400",
  "code": "CLUSTERS-MGMT-400",
  "reason": "Invalid number of compute nodes: 3. Multi AZ clusters require at least 9 compute nodes.",
  "operation_id": "1gaqo99pcf9u3ulqcsa3vsjqvfm8htht"
}
```

I think it should be done in the scope of https://issues.redhat.com/browse/INTLY-10224

Comment (austincunningham) [reply templates/ocm/cluster-template.json:5]: Guess we can get this merged 
/lgtm
/approved

**Review by austincunningham — APPROVED** [review]

File: templates/ocm/cluster-template.json (hunk 2 above)
Comment (austincunningham) [reply templates/ocm/cluster-template.json:5]: /approve

**Pull request merged**